frontend/src/components/Table/Table.tsx

```diff
@@ -372,16 +372,18 @@ const Table = <T extends IIdentifiedObject, TFilter extends object = {}>(
   const renderHeaderCell = (column: ColumnInstanceWithProps<T>) => {
     return (
       <div className="sortable-column">
-        <ColumnFilter
-          onFilter={values => {
-            if (filterFormRef.current?.dirty) {
-              filterFormRef.current.submitForm();
-            }
-          }}
-          column={column}
-        >
-          {column.render('Header')}
-        </ColumnFilter>
+        {filterable ? (
```

Comment:
if columns aren't filterable, don't wrap them in the ColumnFilter component, which requires a formik parent.